inhouse_bot/common_utils/lol_api/tasks.py

```diff
@@ -26,3 +26,27 @@ async def average_win_rate_10_matches(summoner_name: str):
             if participant.puuid == summoner.puuid:
                 wins.append(int(participant.win))
     return statistics.mean(wins or [0])
+
+
+async def get_summoner_by_puuid(puuid: str):
+    summoner = await lol.Summoner(puuid=puuid).get()
+    return summoner
+
+
+async def get_summoner_rank_info_by_id(summoner_id: str):
+    summoner = await lol.SummonerLeague(summoner_id=summoner_id).get()
```

Comment:
Add a "league_v4_summoner_entries": 600 to pyotconf to cache these responses for 600 seconds

Reply:
ok